src/main/resources/assets/computercraft/lua/rom/programs/edit.lua

```diff
@@ -31,12 +31,12 @@ local bRunning = true
 -- Colours
 local highlightColour, keywordColour, commentColour, textColour, bgColour, stringColour
 if term.isColour() then
```

Comment:
So, if bios.lua is going to set colours, it needs to set them differently depending on Advanced or Normal